atd-vze/src/views/Crashes/Crash.js

```diff
@@ -283,6 +293,7 @@ function Crash(props) {
         </Col>
         <Col xs="12" md="6" className="mb-4">
           <CrashDiagram
+            isCOA={isCOA}
```

Comment:
is our state variable down as a prop to the `<CrashDiagram>` component so it can hide the CR3 download button based on it.